model/config.go

```diff
@@ -2698,7 +2698,7 @@ func (s *JobSettings) SetDefaults() {
 }
 
 type CloudSettings struct {
-	CWSUrl *string `access:"environment,write_restrictable"`
+	CWSUrl *string `access:"write_restrictable"`
```

Comment:
Are we sure we want to remove this? Was this confirmed?

Reply:
Yes, I spoke with Maria Nunez. She said it was probably a copy/paste error.